src/pages/SeongHoKim/Login/Login.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import './Login.scss';
+import logo1 from './Wecode_Logo01.png';
+import logo2 from './Wecode_Logo02.png';
+import { useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  // 회원가입 페이지 이동
+  const moveNavigate = useNavigate();
+  const goToSignUp = () => {
+    moveNavigate('/Register');
+  };
+  // input 입력(id)
+  const [id, setID] = useState('');
+  const saveUserID = event => {
+    setID(event.target.value);
+  };
+
+  // input 입력(pw)
+  const [pw, setPW] = useState('');
+  const saveUserPW = event => {
+    setPW(event.target.value);
+  };
+  // ID, PW 조건 : 아이디는 @와 .이 포함되고 비밀번호는 10글자 이상일때
+  const isInvalid = id.includes('@', '.') && pw.length >= 10;
+  // 로그인 버튼 클릭
+  const goToMain = () => {
+    fetch('http://10.58.52.212:8000/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: id,
+        password: pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(data => {
+        if (data.message === 'LOGIN_SUCCESS') {
+          alert('입력값을 확인해 주세요.');
+        } else {
+          alert('로그인 되었습니다.');
+          localStorage.setItem('token', data.message);
+          moveNavigate('/main');
+        }
+      });
+  };
+
+  return (
+    <div className="login_page">
+      <div className="logo_frame">
+        <img className="big_logo" src={logo1} alt="위코드 로고1"></img>
+        <img className="small_logo" src={logo2} alt="위코드 로고2"></img>
```

Comment:
content가 없는 태그에 대해서는 self-closing 사용해주세요
```suggestion
        <img className="big_logo" src={logo1} alt="위코드 로고1" />
        <img className="small_logo" src={logo2} alt="위코드 로고2" />
```

Reply:
넵. self-closing 으로 수정하였습니다.